File: hub-kit-backend/src/main/java/com/inetum/realdolmen/hubkitbackend/dto/PolicyHolderDTO.java
package com.inetum.realdolmen.hubkitbackend.dto;

import lombok.*;

import java.util.List;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class PolicyHolderDTO {
    private Integer id;
    private String firstName;
    private String lastName;
    private String email;
    private String address;
    private String postalCode;
    private String phoneNumber;
    private List<InsuranceCertificateDTO> insuranceCertificates;
}

File: crash-kit-app/app/src/main/java/com/inetum/realdolmen/crashkit/dto/PolicyHolderDTO.kt
package com.inetum.realdolmen.crashkit.dto

data class PolicyHolderDTO(
    val firstName: String? = null,
    val lastName: String? = null,
    val email: String? = null,
    val phoneNumber: String? = null,
    val address: String? = null,
    val postalCode: String? = null,
    val insuranceCertificate: InsuranceCertificate? = null
)